package.json

```diff
@@ -85,18 +87,22 @@
     "url": "https://opencollective.com/storybook"
   },
   "lint-staged": {
-    "*.js": [
-      "npm run lint:js -- --fix",
-      "git add"
-    ],
-    "*.json": [
-      "npm run lint:js -- --fix",
-      "git add"
-    ],
-    "*.md": [
-      "npm run lint:md -- -o",
-      "git add"
-    ]
+    "linters": {
```

Comment:
I had trouble with lint-staged trying to git add multiple things in parallel, which fails because git's lockfile would exist.

I disabled concurrency for this reason. Depending on the amount of files in your stage, this will make the process a second or 2 slower, but git add is reliable.